src/script/search/SearchRepository.ts

```diff
@@ -178,11 +178,11 @@ export class SearchRepository {
    * @note We skip a few results as connection changes need a while to reflect on the backend.
    *
    * @param query Search query
-   * @param isHandle Is query a user handle
+   * @param teamId Current team ID the selfUser is in (will help prioritize results)
    * @param maxResults Maximum number of results
    * @returns Resolves with the search results
    */
-  async searchByName(term: string, maxResults = CONFIG.MAX_SEARCH_RESULTS): Promise<User[]> {
+  async searchByName(term: string, teamId = '', maxResults = CONFIG.MAX_SEARCH_RESULTS): Promise<User[]> {
```

Comment:
Now that we have less team members in the local state, we need to sort remote search results to have the current team members first